ner_v2/detectors/numeral/number/README.md

```diff
@@ -198,10 +201,12 @@ class NumberDetector(BaseNumberDetector):
     data_directory_path = os.path.join((os.path.dirname(os.path.abspath(__file__)).rstrip(os.sep)),
                                        LANGUAGE_DATA_DIRECTORY)
 
-    def __init__(self, entity_name):
+    def __init__(self, entity_name, unit_type=None):
         super(NumberDetector, self).__init__(entity_name=entity_name,
+                                             unit_type=unit_type,
```

Comment:
Move `unit_type` to the end, after `data_directory_path` as `data_directory_path` is a required argument